tests/network/refspecs.c

```diff
@@ -158,3 +158,22 @@ void test_network_refspecs__matching(void)
 
 	git_refspec__free(&spec);
 }
+
+void test_network_refspecs__parse_free(void) {
+	git_refspec *spec;
+	int error;
+
+	spec = NULL;
+
+	error = git_refspec_parse(&spec, "", 0);
```

Comment:
You can get rid of the `error` for this and all the following invocations, as we're always using `cl_git_pass` and `cl_git_fail` to check for errors when the invoked function is conforming to libgit2's error style. So:

```
cl_git_fail(git_refspec_parse(&spec, "", 0));
cl_git_fail(git_refspec_parse(&spec, ":::", 0));
cl_git_pass(git_refspec_pares(&spec, "HEAD:", 1));
```